pytorch_lightning/trainer/trainer.py

```diff
@@ -2095,6 +2095,10 @@ def optimizer_frequencies(self, new_freqs: List[int]) -> None:
 
     @property
     def amp_backend(self) -> Optional[AMPType]:
+        rank_zero_deprecation(
+            "amp_backend is deprecated and will be removed in 1.7. "
+            "Use instance-checks against the PrecisionPlugins directly"
```

Comment:
```suggestion
            "amp_backend is deprecated in v1.6 and will be removed in v1.7. "
            "Use `isinstance` check against the `PrecisionPlugins` directly."
```
are these following one deprecation cycle?